orchagent/vxlanorch.cpp

```diff
@@ -266,6 +317,23 @@ create_tunnel(
         tunnel_attrs.push_back(attr);
     }
 
+    // dest ip
+    if ((dst_ip != nullptr) && p2p)
+    {
+        attr.id = SAI_TUNNEL_ATTR_PEER_MODE;
+        attr.value.s32 = SAI_TUNNEL_PEER_MODE_P2P;
+        tunnel_attrs.push_back(attr);
+        attr.id = SAI_TUNNEL_ATTR_ENCAP_DST_IP;
+        attr.value.ipaddr = *dst_ip;
```

Comment:
If P2P mode used for L3 EVPN, SAI doesn't specify preference of DST_IP from tunnel object or tunnel nexthop object. Which one is used for rewrite??

Reply:
When the SAI_NEXT_HOP_ATTR_TUNNEL_ID refers to a P2P tunnel object, it is expected that the SAI_NEXT_HOP_ATTR_IP and the SAI_TUNNEL_ATTR_ENCAP_DST_IP be the same. The SAI spec doesn't specify 
a preference, however since the p2p tunnel id is being specified instead of a p2mp tunnel id, the p2p tunnel's DIP should be chosen.